script.js

```diff
@@ -1,75 +1,96 @@
+//     humanSelection = e.target.getAttribute("id");
+//     playGame();
+// })
+// scissors.addEventListener("click", (e) => {
+//     computerSelection = getComputerChoice();
+//     humanSelection = e.target.getAttribute("id");
+//     playGame();
+// })
+
 function getComputerChoice() {
-    let cpuChoice;
+
     let choice = Math.floor(Math.random() * 3) + 1;
     
     if (choice === 1) {
-        cpuChoice = "ROCK";
+        return "rock";
     }
     else if (choice === 2) {
-        cpuChoice = "PAPER";
+        return "paper";
     }
     else {
-        cpuChoice = "SCISSORS";
+        return "scissors";
     }
-
-    return cpuChoice;
 }
 
-function getHumanChoice() {
-    let humanChoice = prompt("Rock, Paper, or Scissors?");
-    humanChoice = humanChoice.toUpperCase();
-
-    if(humanChoice === "ROCK" || humanChoice === "PAPER" || humanChoice === "SCISSORS"){
-        return humanChoice;
-    }
-    else {
-        humanChoice = "ROCK";
-        alert("You no try. You don choose rock be that")
-        return humanChoice;
-    }
+function updateScore() {
+    playerScore.textContent = humanScore;
+    cpuScore.textContent = computerScore;
 }
 
 function playGame() {
-    let humanScore = 0;
-    let cpuScore = 0;
-
-    function playRound(humanChoice, cpuChoice) {
-        let message = console.log(`Your choice: ${humanChoice} | CPU choice: ${cpuChoice}`);
-        
+    function playRound(humanChoice, cpuChoice) {        
         if (humanChoice === cpuChoice) {
-            message;
-            console.log("It is a tie")
+            displayMessage.textContent = "It is a tie";
         }
-        else if (humanChoice === "ROCK" && cpuChoice === "PAPER" || 
-        humanChoice === "PAPER" && cpuChoice === "SCISSORS" || 
-        humanChoice === "SCISSORS" && cpuChoice === "ROCK") {
-            message;
-            console.log(`You lose! ${cpuChoice} defeats ${humanChoice}`);
-            cpuScore += 1;
+        else if (humanChoice === "rock" && cpuChoice === "paper" || 
+        humanChoice === "paper" && cpuChoice === "scissors" || 
+        humanChoice === "scissors" && cpuChoice === "rock") {
+            displayMessage.textContent = `You lose! ${cpuChoice} defeats ${humanChoice}`;
+            computerScore += 1;
+            updateScore();
         }
-        else if (humanChoice === "ROCK" && cpuChoice === "SCISSORS" || 
-        humanChoice === "PAPER" && cpuChoice === "ROCK" || 
-        humanChoice === "SCISSORS" && cpuChoice === "PAPER") {
-            message;
-            console.log(`You Win! ${humanChoice} defeats ${cpuChoice}`);
+        else if (humanChoice === "rock" && cpuChoice === "scissors" || 
+        humanChoice === "paper" && cpuChoice === "rock" || 
+        humanChoice === "scissors" && cpuChoice === "paper") {
+            displayMessage.textContent = `You Win! ${humanChoice} defeats ${cpuChoice}`;
             humanScore += 1;
+            updateScore();
```

Comment:
I think there are missing closing brackets and redundant conditions within your if statement, this can cause some errors. Maybe you should combine these conditions into a single block, and ensure brackets are correctly placed.